features/create.feature

```diff
@@ -11,3 +11,19 @@ Feature: upload an app to the apps marketplace
     And the command output should contain "Status  working"
     And the command output should contain "Create  OK"
     And the zip file should exist in directory "tmp/aruba/tmp"
+
+  Scenario: invalid app - missing manifest.json
+    Given an app is created in directory "tmp/aruba"
+    Given I remove file "tmp/aruba/manifest.json"
+    When I run the command "zat create --path tmp/aruba --no-install" to validate the app
+    Then the command output should contain "Could not find manifest.json"
+
+  Scenario: invalid manifest.json - duplicate keys
```

Comment:
The tests below seem redundant, why not only keep the `validate` ones?

Reply:
iirc zat create accesses the [manifest](https://github.com/zendesk/zendesk_apps_tools/blob/master/lib/zendesk_apps_tools/command.rb#L164-L171) before running validate (via [package](https://github.com/zendesk/zendesk_apps_tools/blob/master/lib/zendesk_apps_tools/command.rb#L164-L171)). I added these tests so that if there are problems with manifest, it would cover those scenarios.